### PR TITLE
Support tagging test with type

### PR DESCRIPTION
**Pull request by renfeiw**
- added type in the playlist.xml parsing and make generating logic
- updated target counting logic
- updated documentations
- updated playlist.xsd
- update NativeTest playlist.xml

[ci skip]

Signed-off-by: Renfei Wang <renfeiw@ca.ibm.com>

### CODE REVIEW

**Comment by renfeiw** [comment]
reviewer: @llxia

File: test/README.md [hunk 1]
```diff
@@ -101,17 +101,21 @@ add a line to `TestConfig/resources/excludes/latest_exclude_$(JDK_VERSION).txt`
 
 ## 4) How to execute a different group of tests?
 
-Test can be run with different levels, groups or combination of
-level.group.
+Test can be run with different levels, groups and types or combination of two 
+(i.e., level.group, level.type, group.tyep) or three (i.e., level.group.type)
```
Comment (llxia) [inline test/README.md]: typo `group.type`

File: test/docs/OpenJ9TestUserGuide.md [hunk 2]
```diff
@@ -109,6 +109,7 @@ Please read [DependentLibs.md](./DependentLibs.md) for details.
       - level:   [sanity|extended] (extended default value)
       - group:   [functional|system|openjdk|external|perf|jck] (required 
                  to provide one group per test)
+      - type:    [regular|native] (regular default value)
```
Comment (llxia) [inline test/docs/OpenJ9TestUserGuide.md]: I think we need to explain what does `native` mean.
For example, if a test is tagged with native, it means this test needs to run with test image (native test libs). `NATIVE_TEST_LIBS` needs to be set for local testing. If Grinder is used, native test lib download link needs to be provided in addition to SDK download link in `CUSTOMIZED_SDK_URL`. https://github.com/AdoptOpenJDK/openjdk-tests/wiki/How-to-Run-a-Grinder-Build-on-Jenkins

**Comment by renfeiw** [comment]
updated

File: test/docs/OpenJ9TestUserGuide.md [hunk 3]
```diff
@@ -109,6 +109,13 @@ Please read [DependentLibs.md](./DependentLibs.md) for details.
       - level:   [sanity|extended] (extended default value)
       - group:   [functional|system|openjdk|external|perf|jck] (required 
                  to provide one group per test)
+      - type:    [regular|native] (if a test is tagged with native, it means this
+                 test needs to run with test image (native test libs); 
+                 NATIVE_TEST_LIBS needs to be set for local testing; if Grinder is used, 
+                 native test libs download link needs to be provided in addition to SDK 
+                 download link in CUSTOMIZED_SDK_URL. For details, please refer to 
+                 https://github.com/AdoptOpenJDK/openjdk-tests/wiki/How-to-Run-a-Grinder-Build-on-Jenkins;
```
Comment (llxia) [inline test/docs/OpenJ9TestUserGuide.md]: Could we use the markdown link? For example:

please refer to [How-to-Run-a-Grinder-Build-on-Jenkins](https://github.com/AdoptOpenJDK/openjdk-tests/wiki/How-to-Run-a-Grinder-Build-on-Jenkins)

File: test/docs/OpenJ9TestUserGuide.md [hunk 4]
```diff
@@ -109,6 +109,13 @@ Please read [DependentLibs.md](./DependentLibs.md) for details.
       - level:   [sanity|extended] (extended default value)
       - group:   [functional|system|openjdk|external|perf|jck] (required 
                  to provide one group per test)
+      - type:    [regular|native] (if a test is tagged with native, it means this
+                 test needs to run with test image (native test libs); 
+                 NATIVE_TEST_LIBS needs to be set for local testing; if Grinder is used, 
+                 native test libs download link needs to be provided in addition to SDK 
+                 download link in CUSTOMIZED_SDK_URL. For details, please refer to 
+                 https://github.com/AdoptOpenJDK/openjdk-tests/wiki/How-to-Run-a-Grinder-Build-on-Jenkins; 
+                regular default value)
```
Comment (llxia) [inline test/docs/OpenJ9TestUserGuide.md]: default to regular

**Comment by renfeiw** [comment]
updated

**Comment by llxia** [comment]
jenkins test sanity all jdk8

**Comment by llxia** [comment]
jenkins test sanity all jdk11

**Comment by llxia** [comment]
`cmdLineTester_SCURLClassLoaderNPTests_SE80_1 ` crashed with vmState=0x0005ffff is known #5046

**Pull request merged**